src/components/Tabs.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Box, Button, Divider, Stack} from '@chakra-ui/react';
+
+export const TabContext = React.createContext();
+
+export function Tabs({labels, children}) {
+  const [currentTab, setCurrentTab] = React.useState(labels[0]);
+
+  return (
+    <TabContext.Provider value={currentTab}>
+      <Stack direction="row" spacing="0" wrap>
+        {labels.map(label => (
+          <Box key={label}>
+            <Button
+              color={currentTab === label ? 'navy' : 'gray.400'}
```

Comment:
I believe `navy` is [coming from the brand package](https://github.com/apollographql/docs/blob/37bd9bc5e217fafcd209c78848dc47cc3003684c/src/%40chakra-ui/gatsby-plugin/theme.js#L7), but for some reason it looks off brand in the preview to me. You might have to use the number value?

```suggestion
              color={currentTab === label ? 'navy.400' : 'gray.400'}
```

Reply:
weird, local preview was navy, but hosted preview is gray